### PR TITLE
Sanitize flagging inputs before writing to csv

### PR DESCRIPTION
**Pull request by abidlabs**
To prevent csv injection, this sanitizes flagging inputs before writing to the csv

### CODE REVIEW

File: gradio/utils.py [hunk 1]
```diff
@@ -286,3 +287,35 @@ def get_default_args(func: Callable) -> Dict[str, Any]:
...
+    """Sanitizes data so that it can be safely written to a CSV file."""
+
+    def sanitize(item):
+        return "'" + item
+
+    unsafe_prefixes = ("+", "=", "-", "@")
+
+    if isinstance(data, str):
+        if data.startswith(unsafe_prefixes):
+            warnings.warn("Sanitizing flagged data by escaping cell contents")
```
Comment (FarukOzderim) [inline gradio/utils.py:302]: might be nice to inform user about unsafe_prefixes

**Comment by FarukOzderim** [comment]
LGTM other than the points above, great work as always!

**Comment by abidlabs** [comment]
Awesome, thanks for the helpful suggestions

**Comment by abidlabs** [comment]
I'll hold off on merging until we resolve the big file issue

**Pull request merged**